test/runtests.jl

```diff
@@ -1,52 +1,54 @@
+using ChainRules: Zero
 using Zygote, Test
 using Zygote: gradient
 using CUDA: has_cuda
 
-@testset "Interface" begin
-  include("interface.jl")
-end
-
+@testset "Zygote" begin
```

Comment:
Should these tests be done under one "Zygote" testset? This is the only package I've seen that doesn't do that.

Reply:
This change make me unreasonably happy.